docs/pages/desktop-access/rbac.mdx

```diff
@@ -123,6 +129,79 @@ allow:
   windows_desktop_logins: ["{{internal.windows_logins}}"]
 ```
 
+### Automatic User Creation
+
+<Details title="Version Warning: Before 12.3" opened={false}>
+Teleport versions prior to 12.3 don't support the options shown below.
+</Details>
+
+Teleport's Desktop Service can be configured to automatically create local
+Windows users upon login.
+
+#### Configuration
+
+<Admonition type="warning" title="Local users only">
+Automatic user provisioning is only supported for local users, and does
+not take effect in Active Directory environments.
+</Admonition>
+
+This feature is disabled by default, and can be enabled by setting the
+`create_desktop_user` role option on one of the user's roles.
+
+```yaml
+kind: role
+version: v6
+metadata:
+  name: allow-user-provisioning
+spec:
+  options:
+    create_desktop_user: true
+  allow:
+    windows_desktop_labels: { '*': '*' }
+    windows_desktop_logins: jane
+```
+
+When a Teleport user connects to a desktop, Teleport checks each of the user's
+roles that match the desktop. If at least one role matches the desktop but does
+not include `create_desktop_user: true`, automatic user creation will be
+disabled. Roles that do not match the Node will not be checked.
```

Comment:
```suggestion
disabled. Roles that do not match the desktop will not be checked.
```
Since we introduce the target resource as "desktop". If we want to clarify that this is a Teleport process, maybe we could say "Desktop Service instance" (or "Desktop Service agent")?

Reply:
Thanks, that was a total copy paste job.